main.go

```diff
@@ -212,6 +212,14 @@ func fetchOrCreateTestPlan(ctx context.Context, apiClient *api.Client, cfg confi
 			p := plan.CreateFallbackPlan(files, cfg.Parallelism)
 			return p, nil
 		}
+
+		if billingError := new(api.BillingError); errors.As(err, &billingError) {
+			fmt.Println(billingError.Message)
+			fmt.Println("Using fallback mode. Your build may take longer than usual.")
```

Comment:
I agree with @wooly's comment about being more shouty that this has happened, also maybe the second error line should say something like `Falling back to non-intelligent split, your build may take longer than usual` just so it matches up with marketing copy?